lib/tasks/publish.rake

```diff
@@ -0,0 +1,13 @@
+# This tasks will publish documents that are in a draft publication state.
+
+require "services"
+
+namespace :publish do
```

Comment:
I think we should call this something different because it's not quite doing the same as [pressing "Publish" in the app would do](https://github.com/alphagov/specialist-publisher/blob/master/app/services/document_publisher.rb) (it doesn't send emails or set the update type.

I'd suggest calling it `bulk_imports:publish` since it's closed related to the other bulk import task.

Reply:
no, this actually does exactly the same: https://github.com/alphagov/specialist-publisher/blob/master/app/controllers/documents_controller.rb#L64 as you can see here

Reply:
that service is called by: https://github.com/alphagov/specialist-publisher/blob/master/app/models/document.rb#L237 it just triggers it

Reply:
Oh yes you're right, I must've misread something!